src/static/styles/main.scss

```diff
@@ -79,6 +79,10 @@ h6,
     color: $black;
 }
 
+.form-row .col-no-label {
```

Comment:
Could you use a Bootstrap's `pt-<number>` instead?

Reply:
pt-4 (the maximum) is 1.5em